### DISCUSSION TITLE
options in YAML not changing output

### DISCUSSION THREAD

**Discussion by jkylearmstrong-temple** [quarto-dev/quarto-cli#6920] [Q&A]
### Bug description

The options I put into the YAML are not rending in the HTML in R 

### Steps to reproduce

The code is below:

````
---
title: "title"
subtitle: "subtitle"
author: First Last
date: '`r Sys.Date()`'
date_format: medium
output:
  html_document:
    toc: true
    toc_depth: 6
    number_sections: true
    toc_float:
      collapsed: true
      smooth_scroll: true
---

# Heading 1

```{r}
library('dplyr')

mtcars  %>%
  glimpse()
```

# Heading 2

```{r}
library(ggplot2)

mtcars %>%
  ggplot(aes(x=cyl, y=mpg)) + 
  geom_point()

```
````

### Expected behavior

The date is still long instead of medium, there is no TOC, the TOC is not floating, ect.

### Actual behavior

![image](https://github.com/quarto-dev/quarto-cli/assets/145395480/d2b50246-a6bf-4033-ae48-dee394be30ea)

### Your environment

R version
```
> R.version
               _                                
platform       x86_64-w64-mingw32               
arch           x86_64                           
os             mingw32                          
crt            ucrt                             
system         x86_64, mingw32                  
status                                          
major          4                                
minor          3.1                              
year           2023                             
month          06                               
day            16                               
svn rev        84548                            
language       R                                
version.string R version 4.3.1 (2023-06-16 ucrt)
nickname       Beagle Scouts 
```

R Studio
```
RStudio 2023.06.2+561 "Mountain Hydrangea" Release (de44a3118f7963972e24a78b7a1ad48b4be8a217, 2023-08-25) for windows
Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) RStudio/2023.06.2+561 Chrome/110.0.5481.208 Electron/23.3.0 Safari/537.36
```

System
```
> Sys.info()
       sysname        release        version       nodename        machine          login           user effective_user 
     "Windows"       "10 x64"  "build 22621"  "CBE-68VLWM3"       "x86-64"     "tus75918"     "tus75918"     "tus75918" 
```

R quarto package
```
quarto::quarto_version()
[1] ‘1.3.450’
````

### Quarto check output

>quarto check
A new release of Deno is available: 1.28.2 → 1.36.4 Run `deno upgrade` to install it.

[>] Checking versions of quarto binary dependencies...
      Pandoc version 3.1.1: OK
      Dart Sass version 1.55.0: OK
[>] Checking versions of quarto dependencies......OK
[>] Checking Quarto installation......OK
      Version: 1.3.450
      Path: C:\Program Files\Quarto\bin
      CodePage: 1252

[>] Checking basic markdown render....OK

[>] Checking Python 3 installation....(None)

      Unable to locate an installed version of Python 3.
      Install Python 3 from https://www.python.org/downloads/

[>] Checking R installation...........OK
      Version: 4.3.1
      Path: C:/Users/tus75918/AppData/Local/Programs/R/R-4.3.1
      LibPaths:
        - C:/Users/tus75918/AppData/Local/Programs/R/R-4.3.1/library
      knitr: 1.44
      rmarkdown: 2.25

[>] Checking Knitr engine render......OK

**Comment by mcanouil** [comment]
You are not using Quarto options but RMarkdown, please read the guide (<https://quarto.org/docs/output-formats/html-basics.html>) and the reference page for valid HTML option (<https://quarto.org/docs/reference/formats/html.html>).
